### PR TITLE
Fix Karma Clear Animation & Undo Tests

### PR DESCRIPTION
**Pull request by embarks**
## Context
- Deleted/skipped some tests which were flaking out in #7811, want to add them back but not flaky 💅 

## Summary
We were running these tests which selected the animation from the dropdown, sometimes applied some filters and clicked "undo" and then checked that those filters and animations were reapplied. These tests were all flaky for every element type, but especially background/foreground images and videos.
I believe this was because of a behavior of the animation dropdown. When you select an animation, it plays the animation on the editor canvas. For a brief period there is an animation playing state. If you click the "reset" button _during_ this state, it sometimes does not clear the animations, so we would get `Expected 0 to equal 1` in reference to our test expectation checking how many animations had been applied. We would also see that the Undo button would not appear.

So the biggest change I added to each of the tests was a new step to click the `Stop Page Animations` button right after selecting an animation and applying the filters. 

I believe this was the problem before because we had particular trouble with the media elements, shapes and text were more stable... This was reflected when I did the test clicks manually super fast like the test bot would. 

Here's a replication of what I believe the test was being flaky about:
https://user-images.githubusercontent.com/41136059/120729620-76aa2c80-c494-11eb-9c8b-ec06dfdb832d.mp4
https://user-images.githubusercontent.com/41136059/120729613-7316a580-c494-11eb-812b-7445dbc78170.mp4

So you can see the undo button didn't appear for the image and video, but did for the shape/text. I did see failures for both of those at some point, so I kept the step to cancel the animation, which worked for media.

Here's essentially what the test does now:
https://user-images.githubusercontent.com/41136059/120729883-1cf63200-c495-11eb-89d2-07036261807b.mp4

## Relevant Technical Choices
- I also added "filters" for the foreground/background elements and checked those were properly added, cleared, and reverted
- I left "text" to just be animations
- I checked that the shape's layer opacity would be reset as well
- I renamed the fixture button to resetElementButton for consistency
- I rebased off of Brittany's reset branch so some of her changes are in there :)

## To-do
- I noticed a thing about shape's design panels-- it has two options for opacity. The layer opacity and the "style" opacity. Seems weird. Seems like we should clear the "style opacity" and get rid of "layer opacity" for shapes. 
- I noticed a thing about the undo button-- we should probably clear the old snackbar? the "undo" stacks but they don't properly undo what they said they would.

### QA

- [x] This is a non-user-facing change and requires no QA

### UAT

- [x] UAT should use the same steps as above.

### Does this PR have a security-related impact?
no

### Does this PR change what data or activity we track or use?
no

### Does this PR have a legal-related impact?
no

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

This fix isn't tracked yet.

### CODE REVIEW

**Review by maxyinger — APPROVED** [review]
nice. makes sense and lgtm

File: assets/src/edit-story/app/highlights/quickActions/constants.js [hunk 1]
```diff
@@ -56,9 +55,22 @@ export const ACTION_TEXT = {
...
+export const RESET_DEFAULTS = {
+  TEXT_BORDER_RADIUS: {
+    locked: true,
+    topLeft: 2,
+    topRight: 2,
+    bottomLeft: 2,
+    bottomRight: 2,
+  },
+  STANDARD_BORDER_RADIUS: 0,
 };
```
Comment (samwhale) [inline assets/src/edit-story/app/highlights/quickActions/constants.js:76]: 👍 dope

**Review by samwhale — APPROVED** [review]
Nice this is awesome

**Review by BrittanyIRL — APPROVED** [review]
Awesome!! 
This `I believe this was because of a behavior of the animation dropdown. When you select an animation, it plays the animation on the editor canvas.` makes a lot of sense once you mention it! 

I noticed the shapes having 2 opacity settings as well, I intentionally only reset the layer's opacity because the other one is inline which was kinda where I drew the line for what all we're resetting. Just to shed light on that a bit. I don't understand fully why there's 2 opacity settings there. Probably a good thing to discuss with Prometheus at some point to get some clarity.

**Comment by BrittanyIRL** [comment]
@googlebot I consent.

**Pull request merged**